pontus_description/models/pontus/components/sensors.xacro

```diff
@@ -18,7 +18,7 @@
     <gazebo reference="camera_${camera_id}">
       <sensor name="camera_${camera_id}" type="camera">
         <camera>
-          <horizontal_fov>1.047</horizontal_fov>
+          <horizontal_fov>1.20428</horizontal_fov>
```

Comment:
Where did you get this number? It's definitely closer than the one I originally put but I'm not sure it matches the oakd

Reply:
https://shop.luxonis.com/products/oak-d?srsltid=AfmBOopSnIn7--DWv3Nyrx8JXD2pZj5haGXN7MH7RJr1MXEPkIy2Gezz 

Under technical specifications, it says 69 degrees = 1.20428 radians HFOV for the color camera.